src/bcra_connector/timeout_config.py

```
@@ -1,18 +1,19 @@
 """Timeout configuration for API requests."""
 
 from dataclasses import dataclass
-from typing import Union, Tuple
+from typing import Tuple, Union
```

Ruff

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)